patches/components-content_settings-core-browser-content_settings_pref_provider.cc.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/components/content_settings/core/browser/content_settings_pref_provider.cc b/components/content_settings/core/browser/content_settings_pref_provider.cc
+index 9ffac847adcfb667277910430579d007be7cba09..b9d9f014de464bd4dd6cb5f4e7c63ab40d8885aa 100644
+--- a/components/content_settings/core/browser/content_settings_pref_provider.cc
++++ b/components/content_settings/core/browser/content_settings_pref_provider.cc
+@@ -49,8 +49,6 @@ const char kObsoleteFullscreenExceptionsPref[] =
+ #if !defined(OS_ANDROID)
+ const char kObsoleteMouseLockExceptionsPref[] =
+     "profile.content_settings.exceptions.mouselock";
+-const char kObsoletePluginsExceptionsPref[] =
```

Comment:
I fixed this by copying the old pref value to a different pref in BraveProfileManager